src/octoprint/server/api/settings.py

```diff
@@ -124,6 +124,8 @@ def getSettings():
             "snapshotUrl": s.get(["webcam", "snapshot"]),
             "snapshotTimeout": s.getInt(["webcam", "snapshotTimeout"]),
             "snapshotSslValidation": s.getBoolean(["webcam", "snapshotSslValidation"]),
+            "snapshotAuthUser": s.get(["webcam", "snapshotAuthUser"]),
+            "snapshotAuthPass": s.get(["webcam", "snapshotAuthPass"]),
```

Comment:
This would make it more of a pain to support but I wonder if the password should not be exposed in the UI but instead handled the same way that the GitHub token is. You can set it in the UI, but you can't read it back.